docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md

```diff
@@ -0,0 +1,166 @@
+# AutoMM Detection - Inference with Pretrained Faster R-CNN on VOC Format Dataset
+:label:`sec_automm_detection_infer_coco`
+
+In this section, we show an example to run inference COCO dataset in COCO Format. 
```

Comment:
I think the title, label, the introduction is a bit conflict. 

1. In the title, it says VOC format data, so I assume it is either VOC dataset or some other customized dataset.
2. Then in the label, it says infer_coco
3. In the introduction, it says run inference on COCO in COCO format, which is different from the title.
4. The list below is also a bit confusing, sometimes VOC, sometimes COCO.
5. Then I read the tutorial, I think it is on COCO dataset, but then in the end when the results are shown, the classes are 20 from VOC. 

Can you do a bit rewording here to reduce ambiguity? Thank you.

Reply:
Yea I think I've copied things around and might have messed up with the titles!